app/models/profile.rb

```diff
@@ -32,6 +32,7 @@ class Profile < VersionedModel
 
   has_many :moves, dependent: :nullify
   has_one :person_escort_record
+  has_one :youth_risk_assessment
```

Comment:
This relationship confuses me now.. I was under the impression that the relationship between PER and Profile was purely historic, and that PER's should always now belong to move. Also that PER's don't get reused across moves (which is why we have prefill functionality)... so this relationship seems to be inconsistent as we can have a profile with many moves.. 🤔 Am I missing something in my understanding?

Reply:
I think it would be cleaner for a youth risk assessment to belong solely to a move, and have no direct relationship with a profile - we can always obtain the profile via the move if needed. Perhaps we can discuss over a call...

Reply:
Discussed over slack and agreed that this is fine, if not immediately obvious. An assessment belongs to a move and a profile. If the profile is then used for a subsequent move, the assessment can be accessed via move.profile.assessment, but the assessment will still refer to the original move. Agreed to document this behaviour in the Github wiki as it may not be obvious to clients of the API.

Reply:
updated https://github.com/ministryofjustice/hmpps-book-secure-move-api/wiki/Person-Escort-Record#reuse-the-per-for-a-different-move if you want to have a look

Reply:
Thanks for updating the wiki, it's clear now 👍